src/content/structured/get-started/releases-versioning.mdx

```diff
@@ -72,6 +72,40 @@ We consider Beta components acceptable for use in production-ready applications,
 
 We regularly review the stability and reliability of Beta components. When the component meets the criteria it can be transitioned to a stable release.
 
+## Canary components
+
+Canary components are unstable components in our versioning system that are released for testing purposes.
+
+These components should not be used in production apps without understanding the risk that unexpected changes may be made to them, causing changes within the applications using them.
+
+We consider a canary component acceptable for moving to Beta once customers have tested its usage and provided enough feedback suggesting they're happy with it.
+
+Canary components can be installed in the same way as normal `@ukic/` packages, but prefixed with canary e.g `@ukic/canary-react` or `@ukic/canary-web-components`. You can update to the latest canary version just as you would with normal `@ukic/` packages.
+
+A difference between canary packages and normal `@ukic/` packages is that canary package versions will follow a `<version>-canary.<build-number>` pattern.
+
+<p>
+  Another difference is that docs for canary components won't appear in the
+  components guidance section of this site. Instead, docs can be viewed in our
```

Comment:
Perhaps this should be capitalised as it is referring to the actual section on the site? I.e. which is alongside Styles etc.
```suggestion
  Components guidance section of this site. Instead, docs can be viewed in our
```